packages/archetype/archetype.0.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "The Archetype development team <archetype-dev@edukera.com>"
+authors: ["Benoit Rognier <benoit.rognier@edukera.com>"
+          "Guillaume Duhamel <guillaume.duhamel@edukera.com>"
+          "Pierre-Yves Strub <pierre-yves.strub@polytechnique.edu>"]
+homepage: "https://github.com/edukera/archetype-lang"
+bug-reports: "https://github.com/edukera/archetype-lang/issues"
+dev-repo: "git+https://github.com/edukera/archetype-lang.git"
+license: "MIT"
+
+build: [
+  ["dune" "build"]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
`-p` should always be used for releases.

Reply:
ok thank you for the suggest